src/Storages/LiveView/LiveViewBlockOutputStream.h

```diff
@@ -11,13 +12,18 @@ namespace DB
 class LiveViewBlockOutputStream : public IBlockOutputStream
 {
 public:
-    explicit LiveViewBlockOutputStream(StorageLiveView & storage_) : storage(storage_) {}
+    explicit LiveViewBlockOutputStream(StorageLiveView & storage_, const Context & context) : storage(storage_)
```

Comment:
Probably, we can pass ready stream inside constructor instead of context. It will be more general.

Reply:
I agree. For now I would propose to leave it as is and have this as a task for future code refactoring.